src/Apps/Artist/Routes/Overview/Components/ArtworkFilter/index.tsx

```diff
@@ -76,48 +76,56 @@ class Filter extends Component<Props> {
         </Flex>
 
         <Toggle label="Medium" expanded={!this.showZeroState}>
-          {this.renderCategory({
-            filterState,
-            category: "medium",
-            counts: mediumAggregation.counts,
-            mediator,
-          })}
+          <Flex flexDirection="column" alignItems="left" mb={1}>
+            {this.renderCategory({
+              filterState,
+              category: "medium",
+              counts: mediumAggregation.counts,
+              mediator,
+            })}
+          </Flex>
         </Toggle>
         <Toggle
           expanded={filterState.state.partner_id && !this.showZeroState}
           label="Gallery"
         >
-          {this.renderCategory({
-            filterState,
-            category: "partner_id",
-            counts: galleryAggregation.counts,
-            mediator,
-          })}
+          <Flex flexDirection="column" alignItems="left" mb={1}>
+            {this.renderCategory({
+              filterState,
+              category: "partner_id",
+              counts: galleryAggregation.counts,
+              mediator,
+            })}
+          </Flex>
         </Toggle>
 
         <Toggle
           expanded={filterState.state.partner_id && !this.showZeroState}
           label="Institution"
         >
-          {this.renderCategory({
-            filterState,
-            category: "partner_id",
-            counts: institutionAggregation.counts,
-            mediator,
-          })}
+          <Flex flexDirection="column" alignItems="left" mb={1}>
+            {this.renderCategory({
+              filterState,
+              category: "partner_id",
+              counts: institutionAggregation.counts,
+              mediator,
+            })}
+          </Flex>
         </Toggle>
         <Toggle
           expanded={
             filterState.state.major_periods.length > 0 && !this.showZeroState
           }
           label="Time period"
         >
-          {this.renderCategory({
-            filterState,
-            category: "major_periods",
-            counts: periodAggregation.counts,
-            mediator,
-          })}
+          <Flex flexDirection="column" alignItems="left" mb={1}>
```

Comment:
Each of these `<Flex>` wrappers can be consolidated in a single spot inside of `renderCategory()` above line 142

Reply:
So its like:

```js
return (
  <Flex ...>
    {counts.map( ... )} 
  </Flex>
)
```

Reply:
oooo that is so much nicer. updated!